verification/spec/relayer/relayer.tla

```diff
@@ -0,0 +1,225 @@
+------------------------------ MODULE relayer ------------------------------
+
+EXTENDS Naturals, FiniteSets
+
+CONSTANTS MaxHeight, \* maximal height of all the chains in the system 
+          NrChains \* number of chains in the system
+
+VARIABLES chains, \* a function that assigns to each chainID a chain record
```

Comment:
It seems that the relayer does not (and should not) update `chains`. Can we remove this and provide `Env!<getter>` for all chain queries needed by the relayer? (see below)

Reply:
That's a good point, thanks!